app/models/company.rb

```diff
@@ -17,4 +17,5 @@ def d3_coordinates
     slice(:longitude, :latitude)
   end
 
+  STATUS_OPTIONS = { "Uncontacted" => "uncontacted", "Contacted" => "contacted", "Confirmed" => "confirmed", "Dead" => "dead" }
```

Comment:
This sets the values for the select drop menu on the company show page. The capitalized word is how it displays on the drop menu, and the lowercase is how we store it in the table. If this is too ugly for anyone else, I can figure out how to use an array for this and we can just store capitalized words in the company status column.